platforms/nuget/build-nuspec.py

```diff
@@ -0,0 +1,130 @@
+# run script
+# .\build-nuspec.py --package_name "OpenCV-CPP" --package_version "0.0.1" --output_path "C:\opencv-cpp-build" --sources_path "D:\OpenCV Nuget Windows\C++ nuspec test 1\opencv-nuget" --targets_path "C:\opencv-cpp-build" --targets_file "opencv-cpp.targets"
+
+import os, sys, argparse
+
+authors = 'OpenCV'
+owners = 'OpenCV'
+description = 'This is description.'
+copyright = '\xc2\xa9 ' + 'OpenCV. All rights reserved.'
+tags = 'OpenCV Computer Vision'
+iconUrl = 'https://avatars.githubusercontent.com/u/5009934'
+project_url = 'https://github.com/opencv/opencv'
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="OpenCV CPP create nuget spec script ",
+                                     usage='')
+    # Main arguments
+    parser.add_argument("--package_name", required=True, help="Package name. e.g.: OpenCV.CPP")
+    parser.add_argument("--package_version", required=True, help="Package version. e.g: 1.0.0")
+    parser.add_argument("--output_path", required=True, help="Nuget packages output directory - this is where target, nuspec, etc go.")
+    parser.add_argument("--sources_path", required=True, help="OpenCV source code root.")
+    parser.add_argument("--targets_path", required=True, help="Path name for the generated targets file.")
+    parser.add_argument("--targets_file", required=True, help="File name for the generated targets file.")
```

Comment:
I propose to make list of supported Visual Studio versions as command line argument with default value. It allows to make experiments with single MSVS version and tune config on CI without code modification.

Reply:
Agreed.

This has now been updated to provide the vc version from the nuget action itself.
Here's how it shows up on Github actions:
<img width="419" alt="Screenshot 2021-08-14 at 12 56 12 PM" src="https://user-images.githubusercontent.com/11910885/129438737-e71b91a4-545f-4ac2-ae02-339b6a062484.png">